lib/linters.js

```diff
@@ -225,6 +225,72 @@ function lintErrorFromJavascript(htmlNode, javascriptNode, href, message) {
   return new LintError(href, newLoc, message);
 }
 
+function lintBindingExpression(parsed,
+                               properties,
+                               href,
+                               node,
+                               javascriptNode,
+                               expressionName,
+                               is,
+                               implicitBindings,
+                               name) {
+  var undeclaredBindings = [];
+  parsed.methods.forEach(function(method){
+    // True if the method patches a property.
+    var foundDefinition = false;
+    // True if the matched property is a method
+    var foundMethod = false;
+    properties.forEach(function(property) {
+      if (method == property.name) {
+        foundMethod = property.type == 'Function';
+        foundDefinition = true;
+      }
+    });
+    if (!foundDefinition || !foundMethod) {
+      var message;
+      if (!foundMethod && foundDefinition) {
+        message = name + " using property '" + method + "', " +
```

Comment:
The wording in this message seems a bit confusing due to using "property" and "function" in the same sentence.

Maybe something like "Computed Binding method foo is not defined on element x-bar"?